site/content/documentation/tutorials/getting-started.md

```diff
@@ -434,25 +444,27 @@ for (Statement st: aboutVanGogh) {
 Set<Value> paintings = model.filter(vanGogh, EX.CREATOR_OF, null).objects();
 ```
 
-TIP: Notice that we are suddenly using a new vocabulary constant for our property: `EX.CREATOR_OF`. It is generally a good idea to create a class containing  constants for your own IRIs when you program with RDF4J: it makes it easier to reuse them and avoids introducing typos (not to mention a lot of hassle if you later decide to rename one of your resources). See {{< example "the EX vocabulary class" "model/vocabulary/EX.java" >}} for an example of how to create your own vocabulary classes.
+{{< info >}}
+Notice that we are suddenly using a new vocabulary constant for our property: EX.CREATOR_OF. It is generally a good idea to create a class containing  constants for your own IRIs when you program with RDF4J: it makes it easier to reuse them and avoids introducing typos (not to mention a lot of hassle if you later decide to rename one of your resources). See {{< example "the EX vocabulary class" "model/vocabulary/EX.java" >}} for an example of how to create your own vocabulary classes.
+{{< / info >}}
 
 Once we have selected the values, we can iterate and do something with them. For example, we could try and retrieve further information about each value, like so:
 
 ```java {linenos=inline}
 for (Value painting: paintings) {
-	if (painting instanceof Resource) {
-		// our value is either an IRI or a blank node. Retrieve its properties and print.
-		Model paintingProperties = model.filter((Resource)painting, null, null);
-
-		// write the info about this painting to the console in Turtle format
-		System.out.println("--- information about painting: " + painting);
-		Rio.write(paintingProperties, System.out, RDFFormat.TURTLE);
-		System.out.println();
-	}
+  if (painting instanceof Resource) {
+    // our value is either an IRI or a blank node. Retrieve its properties and print.
+    Model paintingProperties = model.filter((Resource)painting, null, null);
+
+    // write the info about this painting to the console in Turtle format
+    System.out.println("--- information about painting: " + painting);
+    Rio.write(paintingProperties, System.out, RDFFormat.TURTLE);
+    System.out.println();
+  }
```

Comment:
Do we have an example with pretty printing? I remember it took me ages to discover pretty printing, like 2 years actually.

Reply:
We don't, at least not in this tutorial. I am thinking that (outside of the tutorial) we may want to adde more code examples to the examples/ directory in Github, and make those examples easier to import and run for people.